pkg/db/sqlite/sqlite.go

```diff
@@ -215,6 +215,18 @@ var headerFieldToColumnMap = map[string]string{
 	"value": "value",
 }
 
+func (c *Client) ClearRequestLogs(ctx context.Context) error {
```

Comment:
We need to (for now) manually check if there's an open database first, otherwise calling methods on `c.db` could panic. E.g. something similar to the other database methods:
```
if c.db == nil {
	return proj.ErrNoProject
}
```